### ISSUE TITLE
Path in SpectralReport is `null`

### ISSUE THREAD

**Issue by aabedraba** [daveshanley/vacuum#373]
I'm running a SpectralReport against Vercel's OpenAPI document: https://openapi.vercel.sh/

Found the following error: 

```
{
  code: 'duplicated-entry-in-enum',
  path: null,
  message: 'enum contains a duplicate: preview',
  severity: 0,
  range: {
    start: { line: 5373, character: 37 },
    end: { line: 5373, character: 37 }
  },
  source: '/var/folders/5d/df4_xr116p53cxjgfcqd42tr0000gn/T/0a7ba2cf-6c1a-4c8c-a71a-36384ae72ac1.json'
}
```

In the error `path` is `null`, which shouldn't be the case(?): https://github.com/daveshanley/vacuum/blob/main/functions/openapi/duplicated_enum_entry.go#L48

Also, the Typescript interface of Spectral does not expect a null value: https://github.com/stoplightio/spectral/blob/1d3844649ac5ccd833902ce7a84c4633bef842dd/packages/core/src/types/spectral.ts#L14

Worth mentioning, that I ran against over 100 OpenAPI file and this is the only instance where I found this issue. 

WDYT? Bug or expected?

**Comment by daveshanley** [comment]
Looks like a bug to me, not sure yet, will investigate.

**Comment by daveshanley** [comment]
This has been fixed in `v0.7.0`
